inc/functions.php

```diff
@@ -27,10 +33,10 @@ function al_enqueue_scripts() {
  * @return string the modified title
  */
 function al_add_dashes( $title ) {
-	$settings = Post_Modifier_Settings::getInstance();
-	if ( $settings->getSetting( 'plugin_state' ) === 'on' ) {
+
+	if ( getSetting('plugin_state') === 'on' ) {
```

Comment:
I suggest making another function ( `is_active()` or something like that ) which returns `getSetting('plugin_state') === 'on'`